README.md

```diff
@@ -67,3 +67,14 @@ _run_pstd_bscan.m_
 
 This script will generate the input to the executable, run the executable and 
 display sample output.
+
+
+#### Parallelism
+
+TDMS is paralleled with [OpenMP](https://en.wikipedia.org/wiki/OpenMP). The maximum 
```

Comment:
I'm not a native English speaker, but I usually read "[parallelized](https://en.wiktionary.org/wiki/parallelize#English)" (or "parallelised" for the British spelling) when talking about computer programs

Reply:
Which is the same word you used above :sweat_smile:

Reply:
Typo – thanks for spotting 👍🏼